scripts/mk-ros.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -ex
+
+ros="$1"
+
+ARCH=${ARCH:?"ARCH not set"}
+VERSION=${VERSION:?"VERSION not set"}
+
+cd $(dirname $0)/..
+
+strip_bin=$(which strip)
+if [ "${ARCH}" == "arm" ]; then
+  export GOARM=6
+  export CC=/usr/bin/arm-linux-gnueabihf-gcc
+  export CGO_ENABLED=1
+  strip_bin=/usr/arm-linux-gnueabihf/bin/strip
+fi
+GOARCH=${ARCH} go build -tags netgo -installsuffix netgo -ldflags "-X github.com/rancher/os/config.VERSION=${VERSION} -linkmode external -extldflags -static" -o ${ros}
+${strip_bin} --strip-all ${ros}
```

Comment:
We've been doing this for awhile and no problems yet :)  It doesn't remove the line info for go panics, so we haven't had any problem debugging.  But yes, this is for file size.